gramps/plugins/importer/importxml.py

```diff
@@ -156,6 +157,17 @@
 INSTANTIATED = 1
 
 
+def to_bool(value):
```

Comment:
Why did you choose to roll your own `to_bool` rather than use `attrs.converters.to_bool()`?

Reply:
Well, attrs is a dependency, and currently `gramps` doesn't have any (and I couldn't figure out an easy way to make `python setup.py build` install it with our custom setup).